tests/util.go

```diff
@@ -55,6 +63,17 @@ func readFile(t *testing.T, p string) string {
 
 // createdSignedArtifact gets the test dir setup correctly with some random artifacts and keys.
 func createdSignedArtifact(t *testing.T, artifactPath, sigPath string) {
```

Comment:
should this really be moved into pgp.go since it calls a PGP-specific signing method?

Reply:
Whoops, good catch.